CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ All notable changes to this project will be documented in this file.
 - Add ability to use `ChatSDK.createChatAdapter()` for `DirectLine` protocol
 - Add `CreateACSAdapter` telemetry event
 - Improve `ChatSDK.createChatAdapter()` with retries using exponential backoff & additional details on failures
+- Add `GetAgentAvailability` SDK method
```

Comment:
my suggestion is to add that the method is exclusively for auth chats

Reply:
updated